### ISSUE TITLE
Remove control inputs from states

### ISSUE THREAD

**Issue by schillic** [JuliaReach/NeuralNetworkAnalysis.jl#89]
Currently the control inputs are modeled as state variables. This requires some boilerplate code and does not seem necessary. I suggest to change this.

**Comment by mforets** [comment]
> Currently the control inputs are modeled as state variables

yes, as they are held constant during the integration (zero right hand side). it is not idea but can we do otherwise? (if we were working with a linear solver the answer is yes, since we can add those in the "input set" part, but with Julia functions `@taylorize function f(...)` i don't know to do in other way than to assemble the variables into the system

**Comment by schillic** [comment]
Ah, I thought the interface was the same as for `DifferentialEquations`. Can we maybe use the `params` argument for that purpose?

**Comment by mforets** [comment]
last time i checked that feature was not available. but `@taylorize` can handle parameters passed in its third argument, see example [here](https://perezhz.github.io/TaylorIntegration.jl/latest/common/) when it does `local μ = param[1]`. so maybe we just need to tweak `@ivp` in some way. (but that may not be sufficient)
